### DISCUSSION TITLE
Configure dhcpcd noipv6rs option

### DISCUSSION THREAD

**Discussion by Selva-Senthilvelan** [rsmarples/dhcpcd#191] [Q&A] [answered]
Hi,
We are using dhcpcd client in IPv6 environment that does not have Router Adversitements(RA), for stateful DHCPv6 option. We would like to configure the dhcpcd client to forgo the RA message parsing and proceed directly to DHCP Discover/Solicit message. 

What is the correct way to configure the dhcpcd client to achieve this result? Should we use the noipv6rs option or configure it differently? 

Thanks in advance for your responses.
Best regards,
Selva

**Comment by rsmarples** [comment, marked as answer]
You would explicity request an address. For example

```
interface eth0
    ia_na
```

Would request a Normal Address via DHCPv6 regardless of RA (or lack thereof).

**Reply by Selva-Senthilvelan** [reply]
Thank you very much @rsmarples! We are able to request a IPv6 address without the RA, using the above configuration. 
Appreciate your help!
